payment/src/on_created/main.py

```diff
@@ -48,3 +50,21 @@ def handler(event, _):
     })
 
     save_payment_token(order_id, payment_token)
+
+    # Generate custom metrics using the Embedded Metric Format
+    # See https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Specification.html
+    print(json.dumps({
+        "paymentCreated": 1,
+        "environment": ENVIRONMENT,
+        "_aws": {
+            # Timestamp is in milliseconds
+            "Timestamp": int(datetime.datetime.now().timestamp()*1000),
+            "CloudWatchMetrics": [{
+                "Namespace": "ecommerce.payment",
+                "Dimensions": [["environment"]],
+                "Metrics": [
+                    {"Name": "paymentCreated"}
+                ]
+            }]
+        }
+    }))
```

Comment:
The `on_completed` name comes from the event (OrderCompleted), should this be `paymentProcessed` instead of `paymentCompleted`?
```suggestion
    print(json.dumps({
        "paymentProcessed": 1,
        "environment": ENVIRONMENT,
        "_aws": {
            # Timestamp is in milliseconds
            "Timestamp": int(datetime.datetime.now().timestamp()*1000),
            "CloudWatchMetrics": [{
                "Namespace": "ecommerce.payment",
                "Dimensions": [["environment"]],
                "Metrics": [
                    {"Name": "paymentProcessed"}
                ]
            }]
        }
    }))
```